src/igs_service.c

```diff
@@ -814,6 +819,11 @@ igs_result_t igsagent_service_call (igsagent_t *agent,
                                  agent->definition->name, agent->uuid,
                                  local_agent->definition->name, service_name,
                                  local_agent->uuid);
+                    zyre_shouts (agent->context->node, agent->igs_channel,
```

Comment:
What is the point of having 2 messages with the same info shouted to 2 different channels?  
I don't see what this message provides that the previous one does not.  
The 2 previous (already existing) messages are also similar but there are here to ensure backward compatibility between v3.0 and v2.5. The first channel (`calls_channel`) has been deprecated with v3.  
Are we introducing these types of changes with this PR?